packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -117,7 +128,8 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 				( getTemplateLock( _selectedBlockClientId ) === 'contentOnly' ||
 				_selectedBlockName === 'core/block'
 					? _selectedBlockClientId
-					: undefined ),
+					: undefined ) ||
+				closestContentOnlySectionBlock,
```

Comment:
My understanding of the changes in this file is that we want to show the "inspector control" of the parent "content-only" block when it exists rather than the leaf block that is being selected.

Are we doing the same for the block toolbar. This also feels like something we want to do consistently for block inspector and block toolbar?

Reply:
I think this change can probably be extracted to a dedicated PR as it can be seen as an improvement to content only blocks. would be good to e2e test as well.

Reply:
That change is incorrect. I've been looking at how to amend it so that:

- you see the `Content` panel [of the nearest "Section"] in the inspector controls. Similar to when setting `templateLock: contentOnly` on a Group block.
- you see the Inspector Controls (contentOnly version) for the _selected_ block

Reply:
> you see the Content panel [of the nearest "Section"] in the inspector controls. Similar to when setting templateLock: contentOnly on a Group block.
> you see the Inspector Controls (contentOnly version) for the selected block

If we want to combine two blocks in the inspector controls, I feel we need some design explorations no? Or do we just dump everything there? It maybe hard for users to understand what's happening no?

Reply:
@richtabor What are your thoughts here? If you select a child block of a section, should we be exposing the _currently selected_ block's inspector controls or just the parent "section" block's controls?

If it's the former how can we combine that with the `Content` panel being shown at all times without causing overwhelm?

Reply:
I think we should be consistent with what happens when you have a `contentOnly` pattern for now.

Wouldn't mind changing it in a separate PR though. Either to use a drill-down UI like we do for the Navigation block or to just get rid of it entirely.

Reply:
[Drill down](https://github.com/WordPress/gutenberg/pull/64923) might work